types/InflationData.md

```diff
@@ -55,7 +55,31 @@ The query id is the keccak256 hash of the query data:
 ```s
 keccak256(queryData)
 ```
+`0xe5aebfa18b69338df426c64bfab35c1168e20c94ea5a355c26599ae34bfb41f9`
```

Comment:
and this as queryId:

`0x389577823bc36cc44f87d115fcf1d681b9a12be480d449e85013a8cd788fc58a`